service/src/main/resources/api/openapi.yml

```diff
@@ -0,0 +1 @@
+../../../../../common/openapi.yml
```

Comment:
Is there a benefit to such linking here? 
Also - see comment on yml and postgres file

Reply:
The link here is for swagger-ui I believe

Reply:
@iliadsh - Yes, this links is for swagger, which goes back to my other recommendation -- 

Swagger expected this to be under service, because the API is owned by the server, not the client library. Hence the suggestion that we put the spec in the original expected location (this file) rather than link it to a workaround location. Again, not a blocker, but strongly recommend it

Reply:
Ah, I see what you mean now. I'm totally open to moving it under service, however the other terra services I've seen do the 'have it as a common resource' pattern so it might be a consistency issue?

Reply:
@iliadsh  Should not be an issue - we are still in the process of standardizing repo structure, and we can gradually change the others.

Reply:
Moved!